src/main/kotlin/com/openlattice/shuttle/MissionParameters.kt

```diff
@@ -13,14 +13,20 @@ data class PostgresConfiguration(
         @JsonProperty("config") val config: Properties
 )
 
+data class AuthConfiguration(
+        @JsonProperty("credentials") val credentials: Properties
+)
+
 @ReloadableConfiguration(uri = "shuttle.yaml")
 data class MissionParameters(
-        @JsonProperty("postgres") val postgres: PostgresConfiguration
+        @JsonProperty("postgres") val postgres: PostgresConfiguration,
```

Comment:
What is the reason behind loading postgres config from shuttle.yaml instead of rhizome.yaml?

Reply:
This is a good question that I'm not 100% sure on the answer to as that code was written before I touched this file. My guess is that the fields have different values (i.e. max pool size, connection time out) in shuttle.yaml and rhizome.yaml

Reply:
Since we use only the one from shuttle.yaml, I would just copy over those values to rhizome.yaml and remove `postgres` entirely from shuttle (unless we will need 2 sets of different postgres configs)